apps/src/music/blockly/FieldSounds.js

```diff
@@ -20,7 +20,7 @@ const FIELD_PADDING = 2;
 class FieldSounds extends GoogleBlockly.Field {
   constructor(options) {
     const currentValue =
-      options.currentValue || MusicLibrary.getInstance().getDefaultSound();
+      options.currentValue || MusicLibrary.getInstance()?.getDefaultSound();
```

Comment:
The changes in this file and `extensions` allow these music blocks to be rendered without a loaded Music Library. This seems preferable to creating an "unknown block" in any context, but is especially important in documentation.